.gitignore

```diff
@@ -5,3 +5,5 @@
 
 /_output
 /vendor
+/_vendor*/
+/vendor.orig/
```

Comment:
Do we need these two entries? Looks like they are a user use case.

Do we need to add any of dep other files like - https://github.com/golang/dep/blob/master/.gitignore

Reply:
Vendor.orig seems to run every time rep ensure runs. The other one seems to be used during migration but it’s harmless to leave here. I could take it out

Reply:
You are right, they are harmless, didn't realize `dep` as creating those.